screen-pop-app/screen-pop/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
+
+### `yarn test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `yarn build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `yarn eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
```

Comment:
This should be a callout using alert blocks (https://apicentral.inindca.com/meta/authoring/alerts)

Reply:
what alert type do you suggest?

Reply:
e.g.

```
:::info
**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
:::
```

However, I'd suggest removing this section about yarn eject entirely. This is a generated boilerplate readme and that section isn't relevant to this project; ejecting a react app is a recipe for endless tech debt.